init.lua

```diff
@@ -12,46 +12,26 @@
 ]]--
 
 
-
-local wp = minetest.get_worldpath() .. "/luscious"
-minetest.mkdir(wp)
-
 local mgp = minetest.get_mapgen_params()
 local chunksize = 16 * mgp.chunksize
 
-local function cmpy(p2, y)
-	if y > 0 then
-		local h1 = p2 % 16
-		local h2 = math.floor(p2 / 16) * 16
-
-		h1 = h1 - math.max(math.min(h1, math.floor(y / 16)), 0)
-		return math.max(1, h1 + h2)
-	else
-		return math.max(1, p2)
-	end
+local function chose_color(pos)
```

Comment:
tabs and spaces again.

When you modify an existing project, you should adopt the code style, not change it.

Reply:
Turn out that  my software was converting every new tab to space.